fraud/src/main/scala/com/statestreet/DataImportStateStreet.scala

```diff
@@ -191,53 +192,73 @@ object DataImportStateStreet {
       } finally {
         val end = System.currentTimeMillis
         val runtimeInMillis = end - start
-        runtimes += s""""${test}": {"median": ${runtimeInMillis}}""" 
+        metrics += (test -> Map("median" -> runtimeInMillis))
       }
     }
 
+    def getThroughput(test: String, countCode: => Long): Unit = {
+      val runtimeInMillis = metrics get test get "median"
+      val count = countCode
+      val throughput = count / (runtimeInMillis * 0.001)
+      metrics(test) += ("throughput" -> throughput)
```

Comment:
Should we indicate the metrics used here? Like `throughput(elements/sec)`

Reply:
in the history server it'll show up as `ops/s`

![Screen Shot 2019-04-23 at 19 20 39](https://user-images.githubusercontent.com/1004527/56624114-0d8d3500-65fd-11e9-9e68-153c0158ba53.png)